watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/tests/test_dye_desalination.py

```diff
@@ -21,26 +21,27 @@
     build,
     set_operating_conditions,
     initialize_system,
+    initialize_costing,
     solve,
     add_costing,
-    display_costing,
+    optimize_operation,
     display_results,
+    display_costing,
 )
 
-
 solver = get_solver()
 
 
-class TestDyeFlowsheet:
+class TestDyewithROFlowsheetwithPretreatment:
```

Comment:
The only major item that I think we should make sure of is whether RO (0D and 1D) is being tested. We should check a couple of basic results to be safe.